app-ios/Modules/Sources/Timetable/InfoChip.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+struct InfoChip: View {
```

Comment:
Hmm... I named it as InfoChip with reference to Material Design's SuggestionChip, which is used in Android app too.
But actually it is not a chip but just a label in this app's usage, so there might be some more appropriate name 🤔
https://github.com/DroidKaigi/conference-app-2023/blob/bde9dd46b10c32cdc95832cb2fe60f35114506a3/feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableList.kt#L83-L107

Reply:
👀 
I think `InfoChip` isn't bad, and I understand what you mean.
Once I think this is okey. If some idea is come, let's fix this name 🙏🏼